pyomo/contrib/doe/doe.py

```diff
@@ -1241,28 +934,18 @@ def _fix_design(self, m, design_val, fix_opt=True, optimize_option=None):
         --------
         m: model
         """
-        # loop over the design variables and time index and to fix values specified in design_val
-        for d, dname in enumerate(self.design_name):
-            # if design variables are indexed by time
-            if self.design_time[d]:
-                for time in self.design_time[d]:
-                    fix_v = design_val[dname][time]
-
-                    if fix_opt:
-                        getattr(m, dname)[time].fix(fix_v)
-                    else:
-                        if optimize_option is None:
-                            getattr(m, dname)[time].unfix()
-                        else:
-                            if optimize_option[dname]:
-                                getattr(m, dname)[time].unfix()
+        for i, name in enumerate(self.design_name):
+            cuid = pyo.ComponentUID(name)
+            var = cuid.find_component_on(m)
+            if fix_opt:
+                #getattr(m, dname)[time].fix(fix_v)    
+                var.fix(design_val[name])
```

Comment:
Let's talk about using the DesignVariable class to store the nominal point and bounds. Would this be helpful for a sensitivity analysis?